csharp/simplify.web/web.csproj

```diff
@@ -1,10 +1,10 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk.Web">
 	<PropertyGroup>
 		<TargetFramework>net7.0</TargetFramework>
-		<LangVersion>11.0</LangVersion>
-		<Version>0.1</Version>
+		<ImplicitUsings>enable</ImplicitUsings>
+		<TieredPGO>true</TieredPGO>
```

Comment:
Seems a cool feature, is it not needed elsewhere ?
@panesofglass

Reply:
I did that by analogy with aspnet-minimal-api/aspnet-mvc :)